packages/turf-boolean-equal/test/false/linear-rings.geojson

```diff
@@ -7,10 +7,10 @@
       "geometry": {
         "type": "LineString",
         "coordinates": [
-          [-53.674866943359375, 28.30135788537988],
+          [-53.674866943359376, 28.30135788537988],
```

Comment:
I think the round trip of JSON.parse/stringify is modifying these which is a little scary. I think the tests all still pass though.

Reply:
Thanks for checking it out.  Modifiying the output values in the formatting step doesn't sound acceptable.

I can also see how this small text matrix before was laid out nicely, and after is non-optimal.  Not sure what it looks like if it gets formatted with the default 80 printWidth instead.
![image](https://github.com/user-attachments/assets/3502e5f2-dc2f-4e59-bc57-2ac0efb118b7)

Reply:
Everything is going to have tradeoffs. But not automatically formatting these things isn't really an option because we can't review PRs cleanly otherwise. Here's prettier with the default printWidth https://github.com/Turfjs/turf/pull/2677